plugins/backstage-plugin-coder/src/testHelpers/setup.tsx

```diff
@@ -244,14 +177,71 @@ export const renderHookAsCoderEntity = async <
             queryClient={mockQueryClient}
             authStatus={authStatus}
           >
-            <EntityProvider entity={mockEntity}>
-              <>{children}</>
-            </EntityProvider>
+            <EntityProvider entity={mockEntity}>{children}</EntityProvider>
           </CoderProviderWithMockAuth>
-        </TestApiProvider>,
-      ),
+        </TestApiProvider>
+      );
+
+      return wrapInTestApp(mainMarkup) as unknown as typeof mainMarkup;
```

Comment:
There are some strange type mismatches in Backstage. `wrapInTestApp` is supposed to give you back a React component, which you should be able to plug into RTL's functions

But the type is slightly off, so even though things won't break at runtime, the compiler will complain